LinuxEX1.txt

```diff
@@ -0,0 +1,197 @@
+#firts exercise
+#a
+
+touch nameFile.txt
+cd nameFile.txt
+echo "antonna" > nameFile.txt
+cat nameFile.txt
+
+#b
+mkdir new
+
+#c
+cp ~/nameFile.txt ~/new
+cat new/nameFile.txt
+
+#d
+gedit ~/AntonNahhas.txt
```

Comment:
instead of using gedit, consider using a command-line text editor like nano to edit the file. This will allow the script to continue executing without requiring manual input or waiting for the text editor to be closed.